src/main/java/pl/byczazagroda/trackexpensesappbackend/dto/WalletModelMapper.java

```diff
@@ -0,0 +1,11 @@
+package pl.byczazagroda.trackexpensesappbackend.dto;
+
+import org.mapstruct.InjectionStrategy;
+import org.mapstruct.Mapper;
+import pl.byczazagroda.trackexpensesappbackend.model.Wallet;
+
+@Mapper(componentModel = "spring")
+public interface WalletModelMapper {
```

Comment:
Please, delete this interface, and next time make sure that you are up to date with develop branch, because ``WalletModelMapper`` already exists in ``mapper`` package. So please, next time take a note of it.